build_directory.py

```diff
@@ -183,19 +183,19 @@ def cli(
 
     # Fix up 3 repos with camo.githubusercontent.com at random
     # https://github.com/simonw/datasette.io/issues/156#issuecomment-1894795880
-    fix_repo_names = [
-        row[0]
-        for row in db.execute(
-            """
-        select full_name from repos
-        where readme_html like '%camo.githubusercontent.com%'
-        and readme not like '%camo.githubusercontent.com%'
-        limit 3
-    """
-        ).fetchall()
-    ]
-    print("Fixing HTML for {}".format(fix_repo_names))
-    repos_to_fetch.extend(fix_repo_names)
+    # fix_repo_names = [
+    #     row[0]
+    #     for row in db.execute(
+    #         """
+    #     select full_name from repos
+    #     where readme_html like '%camo.githubusercontent.com%'
+    #     and readme not like '%camo.githubusercontent.com%'
+    #     limit 3
+    # """
+    #     ).fetchall()
+    # ]
+    # print("Fixing HTML for {}".format(fix_repo_names))
+    # repos_to_fetch.extend(fix_repo_names)
```

Comment:
This was the worst mess: because I lost the old `content.db` I had to build from scratch, but somewhere our build scripts had depended on the previous DB having those `readme` and `readme_html` columns - without those I couldn't bootstrap a new database.

Reply:
Need to figure out how to rebuild those, while not getting busted by GitHub's rate limit if we need to get READMEs for hundreds of repos.